msg/templates/uorb_microcdr/microRTPS_client.cpp.template

```diff
@@ -0,0 +1,221 @@
+@###############################################
```

Comment:
is `msg/templates/uorb/microRTPS_client.cpp.template` still needed?

Reply:
nope, deleted